support-frontend/assets/pages/new-contributions-landing/components/ContributionThankYou/ContributionThankYouSetPassword.jsx

```diff
@@ -41,35 +41,35 @@ function mapDispatchToProps(dispatch: Dispatch<Action>) {
 // ----- Render ----- //
 
 function ContributionThankYouSetPassword(props: PropTypes) {
-  if (props.paymentMethod === DirectDebit && !props.hasSeenDirectDebitThankYouCopy) {
+  const renderDirectDebit = () => {
```

Comment:
this can be a component
`const DirectDebitCopy = () =>`

and then you can call it like 

`<DirectDebit />`

Reply:
This function sets props of this component so I think it makes more sense to keep it like this rather than make it a child component and try to have it manipulate it parent's properties